docs/auction/index.html

```diff
@@ -16,7 +17,8 @@
 			};
 
 			const auctionResults = await fledge.runAdAuction(options, true);
-			await fledge.renderAd('#ad-slot-1', auctionResults, true);
+			console.log({auctionResults});
+			// await fledge.renderAd('#ad-slot-1', auctionResults, true);
```

Comment:
remove the console? uncomment the renderAd?

Reply:
old code